src/components/MyKiva/MyKivaStats.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+	<section class="tw-flex tw-overflow-hidden tw-flex-col tw-justify-center tw-items-center tw-px-16 tw-py-8 tw-bg-green-800 tw-max-md:tw-px-5">
```

Comment:
We need to use colors that are defined in our brand guidelines, and that means not using tailwind color classes like `tw-text-neutral-100` or `tw-bg-green-800`. Instead you can use the color classes we have defined here: . The exact colors you should use are also defined in the Figma design, when you click on the element and look at the "Properties" tab. In this case, I think you should use `tw-bg-eco-green-3` and `tw-text-primary-inverse`.